Source/Tests/Trakt.NET.Objects.Get.Tests/Shows/Json/Reader/ShowCastAndCrewObjectJsonReader/ShowCastAndCrewObjectJsonReader_Json_String_Tests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using FluentAssertions;
```

Comment:
Usings should be inside the namespace, like this:
```csharp
namespace TraktNet.Objects.Get.Tests.Shows.Json.Reader
{
    using FluentAssertions;
    using System.Linq;
    using System.Threading.Tasks;
    using Trakt.NET.Tests.Utility.Traits;
    using TraktNet.Objects.Get.Shows.Json.Reader;
    using Xunit;
```

It doesn't really matter. Just for consistency with the rest of the codebase.